sdk/tracing/src/main/java/io/opentelemetry/sdk/trace/export/BatchSpanProcessor.java

```diff
@@ -160,150 +153,134 @@ public void forceFlush() {
       droppedSpans =
           droppedSpansCounter.bind(
               Labels.of("spanProcessorType", BatchSpanProcessor.class.getSimpleName()));
+      LongCounter exportedSpansCounter =
+          meter
+              .longCounterBuilder("exportedSpans")
```

Comment:
what would you think about having only one counter called "processedSpans", with a boolean label for dropped or not?